pages/linux/wodim.md

```diff
@@ -0,0 +1,24 @@
+# wodim
+
+> Command for record/burning data using a optical drive to CD or DVD.
+> Some usage of wodim can cause destructive actions, such as erasing all the data on a disc.
+
+- Display optical drives avaliable to wodim:
+
+`wodim --devices`
+
+- Display help switches for use with wodim:
+
+`cdrecord`
+
+- Record a pure CD-DA audio:
+
+`wodim dev=/dev/{{optical_drive}} -audio {{track*.cdaudio}}`
+
+- Burn a file to disc in optical drive where the disc will be ejected after recording (this is needed for some devices):
+
+`wodim -eject dev=/dev/{{optical_drive}} -data {{file.iso}}`
+
+- Burn a file to disc in optical drive, with the option to record to the same disc over multiple sessions:
```

Comment:
Again, this feels a bit awkward. Perhaps

```
Burn a file to the disc in an optical drive, potentially writing to multiple discs in succession:
```

I guess it depends on how the multiple sessions bit is worded as to how it's worded here.